test/gleam/string_test.gleam

```diff
@@ -91,6 +92,20 @@ pub fn replace_test() {
   |> should.equal("Gleam++Erlang++Elixir")
 }
 
+pub fn replace_with_regex_test() {
+  use re <- try(regex.from_string("[,]+"))
```

Comment:
This will cause the test to silently pass if the regex doesn't compile, so let's use `let assert Ok(re) =`

Reply:
Ok, fixed